Corona-Warn-App/src/main/res/layout/rat_profile_create_fragment.xml

```diff
@@ -155,8 +155,8 @@
                     android:layout_width="match_parent"
                     android:layout_height="wrap_content"
                     android:focusable="true"
-                    android:imeOptions="actionNext"
-                    android:inputType="text" />
+                    android:imeOptions="actionNext|flagNoPersonalizedLearning"
+                    android:inputType="number" />
```

Comment:
I changed this to numbers @mtwalli Would revert if that was intended to allow letters as well